### PR TITLE
kubelet/get-pods-from-path: correct description of implemention

### PR DESCRIPTION
**Pull request by chentao1596**
**What this PR does / why we need it**:
    I find this description does not follow the current implementation, it should be describe like this according to my understanding of the source code.

### CODE REVIEW

**Comment by chentao1596** [comment]
@yujuhong @vishh @dchen1107 hi, /cc, please, thank you

**Comment by chentao1596** [comment]
@yujuhong @vishh @dchen1107 hi, please take a look, thank you!

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -94,8 +94,9 @@ containers which were not created by Kubernetes.
 Other than from an PodSpec from the apiserver, there are three ways that a container
 manifest can be provided to the Kubelet.
 
-File: Path passed as a flag on the command line. This file is rechecked every 20
-seconds (configurable with a flag).
+File: Path passed as a flag on the command line. Path and files under this path will
+always be watched, and this work will be restarted after 20 seconds(configurable with
```
Comment (vishh) [inline cmd/kubelet/app/server.go]: Suggest "Files under this path will be monitored periodically for updates. The monitoring period is 20s by default and is configurable via a flag. "

Comment (chentao1596) [inline cmd/kubelet/app/server.go]: yes, this expression is more clearly, thank you for your suggest, i have changed it!

**Comment by chentao1596** [comment]
@Random-Liu hi, i have formatted codes because of building fail, i am very sorry for this trouble with you, PTAL, thank you!

**Comment by chentao1596** [comment]
@Random-Liu I have checked this format of codes, but this mistake is still exist, can you help to ask to test it again? thank you!

**Comment by derekwaynecarr** [comment]
@k8s-bot test this

**Comment by brendandburns** [comment]
/lgtm

/release-note-none

LGTM + release-note-none from the bulk LGTM tool

**Comment by brendandburns** [comment]
@k8s-bot ok to test

/lgtm
/approve

**Pull request merged**